recipe/patches/0002-Do-not-link-directly-to-LLVM-static-libraries.patch

```diff
@@ -1,74 +0,0 @@
-From e47b018d855c7f9587e37dc094b34260446967d9 Mon Sep 17 00:00:00 2001
-From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
-Date: Thu, 21 Nov 2024 15:02:04 +0100
-Subject: [PATCH 02/16] Do not link directly to LLVM static libraries
```

Comment:
I think it would be good to leave this patch checked-in (and commented out under `patches:`), if we ever want to change back to dynamic linkage to LLVM.